app/components/works/buttons_component.html.erb

```diff
@@ -23,6 +23,8 @@
     <%= form.submit 'Save as draft', class: 'btn btn-primary', id: 'save-draft-button',
                                      data: { action: 'unsaved-changes#allowFormSubmission' } %>
     <%= form.submit submit_button_label, class: 'btn btn-primary',
+                                         disabled: false,
+                                         'data-deposit-button-target': deposit_button_target,
```

Comment:
This is not a big deal, admittedly, but you could consider putting all this alongside the data attrs in the existing `data` hash arg. So like:

```ruby
data: { action: 'unsaved-changes#allowFormSubmission', deposit_button_target: deposit_button_target }
```

Reply:
I like that better. Done.